Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/BlendColorGradingLuts.azsl

```diff
@@ -49,13 +49,13 @@ ShaderResourceGroup PassSrg : SRG_PerPass_WithFallback
     float m_weight3;
     float m_weight4;
 
-    // Destination LUT
-    RWTexture3D<float4> m_blendedLut;
-    uint3 m_blendedLutDimensions;
-    
     int m_blendedLutShaperType;
     float m_blendedLutShaperBias;
     float m_blendedLutShaperScale;
+
```

Comment:
Meta-comment, I wonder if we should standardize coalescing scalar members within a struct. It's difficult to glance at this SRG in particular and immediately understand the alignment layout

Reply:
Not sure why this change was needed. Tony, was this is more for aesthetic reasons? The loose data is tracked as part of Constant data and has much better protection for alignment.

Reply:
Will check to see if device update also fixes the alignment message on this one.

Reply:
reverted this after verifying on updated android version